test/app-luatest/gh_7479_bug_fix_uri_lib_test.lua

```diff
@@ -0,0 +1,49 @@
+local t = require('luatest')
+local g = t.group()
+local popen   = require('popen')
+local net_box = require('net.box')
+local server  = require('test.luatest_helpers.server')
+
+-- Create test instance.
+g.before_all = function()
+    g.server = server:new({alias = 'test-gh-7479'})
+    g.server:start()
+    -- Get port for connection testing.
+    g.server_port = g.server:eval("return require('console').listen(0):name()['port']")
+end
+
+-- Stop test instance.
+g.after_all = function()
+    g.server:stop()
+end
+
+-- Checks whether it is possible to specify only port for tarantoolctl connect.
+g.test_tarantoolctl_connect = function()
+    local tarantoolctl_location = io.popen('which tarantoolctl'):read()
```

Comment:
We want to test tarantoolctl from the build:

```Lua
local tarantoolctl_location = fio.pathjoin(os.getenv('BUILDDIR'),
                                           'extra', 'dist', 'tarantoolctl')
```

Reply:
Done, but it began to work only with the `tarantoolctl.in` instead of `tarantoolctl`. AFAIU, it is ok?

Reply:
No, it isn't. `tarantoolctl.in` is a source file - it shouldn't be executed. Here's how we find `tarantoolctl` in another test:

https://github.com/tarantool/tarantool/blob/ac5f303d2b65dde0d1fa889ad716b4c7f154846c/test/app-tap/tarantoolctl.test.lua#L16-L31

Reply:
Done. Thanks! It works with the option `--builddir`